noggin/tests/unit/conftest.py

```diff
@@ -40,6 +40,7 @@ def client(ipa_cert):
     app.config['TESTING'] = True
     app.config['DEBUG'] = True
     app.config['WTF_CSRF_ENABLED'] = False
+    app.config['HIDDEN_GROUPS'] = ['ipausers']
```

Comment:
Are you sure you need to override it here? I thought the defaults were loaded for the testing mode too, no?

Reply:
I thought the same, but saw a failure without it. However now I can't reproduce, so maybe it was unrelated. I've removed it now.